gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -79,7 +78,7 @@
   private final ExecutorProvider executorProvider;
   private final HeaderProvider headerProvider;
   private final String endpoint;
-  private final EnvironmentProvider envProvider;
+  private final boolean attemptDirectPath;
```

Comment:
This will break existing tests for DirectPath enabled clients. Can this value be initialized to the state of the environment variable?

Reply:
Yes we can initialize the value to use the env var state, but one thing I'm curious, I thought we are going to remove the env var entirely, no?

Reply:
Yes, but the transition needs to be gradual:
1. add the setter
2. bigtable client pulls in new version and updates tests to use the new setter instead of the env. var
3. env var is removed
4. when bigtable client is ready to set that flag by default, the setting will move from test only to default settings

My understanding is that step 4 will not happen until a new release of grpc is out supporting the new features we discussed in the meeting.